test/jdk/java/util/Formatter/Padding.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright nit: per OpenJDK conventions, the new copyright for the updated file should be "2023, 2024" not just "2024".

Reply:
```suggestion
 * Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
```
with a `,` after the 2nd year as well. Otherwise a check will fail, as I learned the hard-way ;-)